### ISSUE TITLE
Spurious warnings when using CBMC 6.0.0

### ISSUE THREAD

**Issue by rod-chapman** [model-checking/cbmc-viewer#151]
Applying CBMC 6.0.0 to the s2n-tls codebase, I see hundreds of warnings of the form:
```
WARNING: Property name not of the form STRING.INTEGER: s2n_hmac_is_available.no-body.s2n_libcrypto_is_awslc
```

These arrear to stem from the function key() in cbmc_viewer/propertyt.py which looks for a string of the form STRING.INTEGER only.

Is this warning significant? If not, can it be removed?
